plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizardTest.java

```diff
@@ -0,0 +1,15 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StandardProjectWizardTest {
+
+  private StandardProjectWizard wizard = new StandardProjectWizard();
+
+  @Test
+  public void testCanFinish() {
+    Assert.assertTrue(wizard.canFinish());
```

Comment:
Yes, this will have to change (along with a lot of other things). ATM I'm just trying to make something that runs and doesn't require you to cancel out. No actual project is yet created.